### DISCUSSION TITLE
How to access a pane, and set text to it?

### DISCUSSION THREAD

**Discussion by 369315716** [FlaUI/FlaUI#632] [Q&A]
Because of FlaUI doesn't have As Pane method, how to get it and set text value to it? Could someone give me some redirection and guideline? Huge thanks.

![Weixin Screenshot_20240524112904](https://github.com/FlaUI/FlaUI/assets/170694570/f020e285-820c-42a2-a936-d7c3a14f64ee)

**Comment by fernandes-natanael** [comment]
You can do some dirty like click in the pane and then use keyboard.type to type, If the light way is not working

**Comment by 369315716** [comment]
Thank you for your help, I resolve my question
